src/Processors/Transforms/finalizeChunk.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <Interpreters/AggregateDescription.h>
+#include <Core/Block.h>
+#include <Processors/Chunk.h>
+#include <vector>
+
+namespace DB
+{
+
+using ColumnsByPosition = std::vector<bool>;
```

Comment:
this name doesn't really fit here IMO. what about `ColumnsMask` or smth similar? or get back to just a list of indices?

Reply:
Agree, ColumnsMask looks better. Rebased.